src/components/Bar.jsx

```diff
@@ -77,46 +79,58 @@ class Bar extends Component {
     this.setState({displaySupport: !displaySupport})
   }
 
-  render () {
-    const { t, lang, appName,
-      appEditor, iconPath, replaceTitleOnMobile, displayOnMobile,
-      onDrawer, isPublic } = this.props
+  renderCenter () {
+    const { appName, appEditor, iconPath, t, replaceTitleOnMobile, lang } = this.props
+    return <h1 lang={lang} className={`coz-bar-title ${replaceTitleOnMobile ? 'coz-bar-hide-sm' : ''}`}>
+      <img className='coz-bar-hide-sm' src={iconPath} width='32' />
+      {appEditor && <span className='coz-bar-hide-sm'>{appEditor}</span>}
+      <strong>{appName}</strong>
+      <sup className='coz-bar-hide-sm coz-bar-beta-status'>{t('beta')}</sup>
+    </h1>
+  }
+
+  renderLeft () {
+    const { t } = this.props
+    return <button className='coz-bar-burger' onClick={this.toggleDrawer} data-icon='icon-apps'>
+      <span className='coz-bar-hidden'>{t('drawer')}</span>
+    </button>
+  }
```

Comment:
This looks like a functional component. But it's not. Is there a reason you choose this implementation rather than the classical one?

Reply:
I don't understand what do you mean by classical one ? I used sub-renders since I need the `toggleDrawer` methods and the `t` property. To divide the main render in subrenders enables step by step refactoring. Anyway I have done this refactoring in the other PR #107.

Reply:
Je vais le faire en français, ce sera plus clair pour tout les francophones : 
On dirait un composant fonctionnel. Mais en fait non. Est-ce qu'il y a une raison pour laquelle tu as choisi cette implémentation au lieu de celle couramment utilisé ?

```jsx
  renderLeft () {
   const { t } = this.props
   return <button className='coz-bar-burger' onClick={this.toggleDrawer} data-icon='icon-apps'>
     <span className='coz-bar-hidden'>{t('drawer')}</span>
   </button>
 }
```

vs un composant fonctionnel classique : 

```jsx
function renderLeft (props) {
  const { t } = props;
  return (
    <button className='coz-bar-burger' onClick={this.toggleDrawer} data-icon='icon-apps'>
     <span className='coz-bar-hidden'>{t('drawer')}</span>
   </button>
  );
}
```

Reply:
En l'occurence ici, `this.toggleDrawer` et devrait être passé en props. translate() devrait être appliqué au composant fonctionnel aussi. C'est pour ça que j'ai préféré des sub-renders :)